include/deal.II/distributed/tria.h

```diff
@@ -966,6 +975,28 @@ namespace parallel
       (std::map<unsigned int, std::set<dealii::types::subdomain_id> >
        &vertices_with_ghost_neighbors);
 
+      /**
+       * Fills a map that, for each vertex, lists all the processors whose
+       * subdomains are adjacent to that vertex on the given level for the
+       * multigrid hierarchy. Used by DoFHandler::Policy::ParallelDistributed.
+       */
+      void
+      fill_level_vertices_with_ghost_neighbors
+      (const unsigned int level,
+       std::map<unsigned int, std::set<dealii::types::subdomain_id> >
+       &vertices_with_ghost_neighbors);
+
+      /**
+       * This method returns a bit vector of length tria.n_vertices() with the
+       * locally active vertices on a level, i.e., the vertices touched by the
+       * locally owned cells on that level (possibly including the vertices
```

Comment:
can you say something like "locally owned level cells for use in geometric multigrid"?